src/pages/PrivateNotes/PrivateNotesListPage.js

```diff
@@ -64,11 +64,11 @@ function PrivateNotesListPage({report, personalDetailsList, network, session}) {
     const {translate} = useLocalize();
 
     useEffect(() => {
-        if (network.isOffline) {
+        if (network.isOffline && report.isLoadingPrivateNotes) {
```

Comment:
This caused https://github.com/Expensify/App/issues/30246 where the Not Found page shows instead of a skeleton loader, more details [here](https://github.com/Expensify/App/issues/30246#issuecomment-1778492320).